Mathlib/Tactic/Positivity/Basic.lean

```diff
@@ -521,3 +521,9 @@ def evalNegPart : PositivityExt where eval _ _ e := do
     assertInstancesCommute
     return .nonnegative q(negPart_nonneg $a)
   | _ => throwError "not `negPart`"
+
+end Positivity
+
+end Meta
```

Comment:
Also pre-existing: tactics are sometimes in the `Meta` and sometimes in the `Tactic` namespaces: is there an intentional difference? (Presumably also out of scope of this PR.)